### PR TITLE
Added some guidance on widths

### PR DESCRIPTION
**Pull request by wbamberg**
Add some clarifications about designing examples for the likely width of the output pane.

### CODE REVIEW

**Comment by wbamberg** [comment]
@chrisdavidmills , please let me know if this is helpful.

File: CONTRIBUTING-HTML.md [hunk 1]
```diff
@@ -46,17 +46,27 @@ touch td.html
...
 *   Try to show some important attributes
 *   Try to keep the HTML fragment to under 20 lines. If you have to go over, that's fine, but _really_ try to keep it under 30.
 *   Try to keep HTML fragment line length to under 64 characters. If you have to go over, that's fine, but the line will probably then wrap for most screen widths (be aware that the example gets less width when embedded in an MDN page than it does standalone)
 
+Pay attention to how the example will look at narrower widths. When you test the example locally, it gets to occupy the whole browser window. But when embedded in an MDN page the example has to share space with other page elements.
+
+As a rough guide: https://developer.mozilla.org/en-US/docs/User:wbamberg/HTML_editor_user_test_pages/datalist shows what the editor will look like in an MDN page. In that page, with a browser window width of 1440 px, the entire editor gets about 1000px of width, and the output pane gets about 375px. So examples really need to well at this width.
```
Comment (chrisdavidmills) [inline CONTRIBUTING-HTML.md]: Missing word in "So examples really need to well at this width"

**Comment by wbamberg** [comment]
Thanks @chrisdavidmills , PTAL!

**Comment by wbamberg** [comment]
On second thought, let's just merge this.

**Pull request merged**